package.json

```diff
@@ -62,13 +62,16 @@
     "prettier-plugin-package": "^1.3.0",
     "replace-in-file": "^6.3.5",
     "rimraf": "^3.0.2",
-    "rollup": "^3.23.0",
+    "rollup": "^4.4.0",
     "rollup-plugin-terser": "^7.0.2",
     "stylelint": "^14.13.0",
     "stylelint-config-prettier": "^9.0.3",
     "stylelint-config-vaadin": "^0.3.0",
     "typescript": "^4.8.4"
   },
+  "resolutions": {
+    "@open-wc/semantic-dom-diff": "^0.20.1"
```

Comment:
The old version comes through `@vaadin/testing-helpers` and causes duplicate version of `@web/test-runner-commands` to be installed. While it's not a problem, I thought we can pin the version to the latest.